dynawo/sources/API/JOB/xsd/jobs.xsd

```diff
@@ -171,6 +171,7 @@
 
   <xs:complexType name="FinalStateValuesEntry">
     <xs:attribute name="inputFile" use="required" type="xs:string"/>
+    <xs:attribute name="exportMode" use="required" type="dyn:FinalStateValuesExportMode"/>
```

Comment:
I would propose optional and with default value XML

Reply:
I would say CSV is a better default value